spec/namespaces/indices.yaml

```diff
@@ -1860,6 +1860,11 @@ components:
                 $ref: '../schemas/_common.yaml#/components/schemas/Routing'
               search_routing:
                 $ref: '../schemas/_common.yaml#/components/schemas/Routing'
+              is_hidden:
+                x-version-added: '2.16'
+                description: |-
+                  If `true`, the alias will be hidden, defaults to `false`.
+                type: boolean
```

Comment:
Add `default: false`, and no need to say "defaults to ..." in the text. We're going to clean these up at some point.